config/routes/api.rb

```diff
@@ -43,9 +43,9 @@
       end
 
       resources :folders, except: [:new, :edit, :destroy]
-
       resources :api_users, only: [:create, :destroy, :index], module: 'teams'
       resources :members, except: [:new, :edit], module: 'teams'
+      resource :favourites, only: [:create, :destroy], module: 'teams'
```

Comment:
Why resource and not resources?

Reply:
Huh, this seems more of a typo. Interesting that it actually works nonetheless. Thanks for the hint

Reply:
It is not a typo! @Thunderguts13  is simply a genius
https://guides.rubyonrails.org/routing.html#singular-resources

This allows us to use: `DELETE /api/teams/1/favourites`

otherwise we would have needed to use: `DELETE /api/teams/1/favourites/1`

That's really cool :D

Reply:
use api/teams/1/favourite instead of api/teams/1/favourites

it's not possible to have multiple favourites per team ! and in this case it's more an action than resources.